packages/ssz/src/util/types.ts

```diff
@@ -1 +1,8 @@
 export type Require<T, K extends keyof T> = T & Required<Pick<T, K>>;
+
+/** Snapshot for PartialListCompositeType */
```

Comment:
Can you add more comments here?

Eg:
A snapshot contains the minimum amount of information needed to reconstruct a merkleized list, for the purposes of appending more items. Note: This does not contain list elements, rather only contains intermediate merkle nodes.